### DISCUSSION TITLE
Upload images and other media from buffer

### DISCUSSION THREAD

**Discussion by devopg** [imzbf/md-editor-v3#225] [Q&A] [answered]
Hello. How can i do right upload any media from buffer?  example i has image in buffer , and press ctrl+v and image uploaded

**Comment by imzbf** [comment, marked as answer]
1. Bind the event of uploading pictures `onUploadImg`. Here is an [example](https://imzbf.github.io/md-editor-v3/demo#%F0%9F%93%B7%20Upload%20Pictures).
2. Then press `ctrl+v`.

**Reply by devopg** [reply]
sorry for the stupid question, don't know how I missed it in the documentation via ctrl+f search
